src/main/java/net/atomique/ksar/XMLConfig.java

```diff
@@ -28,71 +30,60 @@
 import javax.xml.parsers.SAXParserFactory;
 
 public class XMLConfig extends DefaultHandler {
+  private static final String KSAR_DTD_PREFIX = "-//NET/ATOMIQUE/KSAR/";
```

Comment:
"-//NET/ATOMIQUE/KSAR/"; should be without trailing "-//NET/ATOMIQUE/KSAR" 
otherwise you will have double // within the string "-//NET/ATOMIQUE/KSAR//Config.xml"